pkg/wasm/cache.go

```diff
@@ -72,25 +83,35 @@ type LocalFileCache struct {
 
 var _ Cache = &LocalFileCache{}
 
+type moduleKey struct {
+	// Identifier for the module. It should be neutral for the checksum.
+	// e.g.) oci://docker.io/test@sha256:0123456789 is not allowed.
+	//       oci://docker.io/test@latest (tagged form) is allowed.
```

Comment:
```suggestion
	//       oci://docker.io/test:latest (tagged form) is allowed.
```

Reply:
although in the code looks like we strip off the tag?

Reply:
Yes, just for future changes, I took a note there to keep the intention.